spec/controllers/my/files_controller_spec.rb

```diff
@@ -9,6 +9,13 @@
     end
   end
 
+  let(:other_collection) do
+    Collection.new(title: 'other test collection').tap do |c|
```

Comment:
Oh right, I misread and had the .tap there still which made it behave the other way. I'll fix this in a minute.